quinn-udp/src/unix.rs

```diff
@@ -568,6 +753,17 @@ fn prepare_msg(
     encoder.finish();
 }
 
+#[cfg(all(
+    not(feature = "fast-apple-datapath"),
```

Comment:
I think it would be good for `fast-apple-datapath` to be a no-op on non-apple platforms. As it is now, on linux, with `fast-apple-datapath`, this won't compile.

How about something along the lines of:

```
#[cfg(any(
  target_os = // all non apple platforms,
  all(
        not(feature = "fast-apple-datapath"),
        any( /* all apple platoforms */),
  )
))]